superset-frontend/cypress-base/cypress/integration/explore/visualizations/table.test.ts

```diff
@@ -119,7 +129,7 @@ describe('Visualization > Table', () => {
   it('Test table with percent metrics and groupby', () => {
     const formData = {
       ...VIZ_DEFAULTS,
-      percent_metrics: NUM_METRIC,
+      percent_metrics: PERCENT_METRIC,
```

Comment:
For some weird reason this test was flaking out when run with `NUM_METRIC` (I replicated this test locally, but was not able to make it fail). Switching to `PERCENT_METRIC` below seems to have solved it.